prebuild.sh

```diff
@@ -0,0 +1,8 @@
+# This is a HACKY script to fix build errors that occur when using ethereumjs v6+
+# Please refer to this issue if you don't understand what this is all about:
+# https://github.com/ethereumjs/ethereumjs-monorepo/issues/2295#issue-1377817460
+# Hopefully this will be removed when the issue is properly resolved.
+
+sed -i "s/import AsyncEventEmitter = require('async-eventemitter')/import AsyncEventEmitter from 'async-eventemitter'/g" ./node_modules/@ethereumjs/evm/src/evm.ts
+sed -i 's/export .*/export {   getActivePrecompiles, precompiles,   ripemdPrecompileAddress }\nexport type {   AddPrecompile,   CustomPrecompile,   DeletePrecompile, PrecompileFunc,   PrecompileInput }/g' ./node_modules/@ethereumjs/evm/src/precompiles/index.ts
+sed -i '205,$d' ./node_modules/@ethereumjs/evm/src/precompiles/index.ts
```

Comment:
I think it'd be better to create a `patch` file in the source tree we can apply here instead of relying on sed. Might be cleaner and who know when the new release with the fix will come out.

Reply:
Hi there, I will prepare the new EthereumJS releases today and releases are planned for tomorrow.

Reply:
Release PR: https://github.com/ethereumjs/ethereumjs-monorepo/pull/2361